.ci/scripts/release/github-release.sh

```diff
@@ -1,32 +1,61 @@
 #!/bin/bash -xeu
 
```

Comment:
This file is just exactly what it is on main now. The main thing is the creation of the go client tag, which is moved from here to post-release-go (as in the main branch).